json-smart/src/main/java/net/minidev/json/parser/JSONParserBase.java

```diff
@@ -103,7 +107,7 @@ public JSONParserBase(int permissiveMode) {
 		this.useHiPrecisionFloat = (permissiveMode & JSONParser.USE_HI_PRECISION_FLOAT) > 0;
 		this.checkTaillingData = (permissiveMode & (JSONParser.ACCEPT_TAILLING_DATA
 				| JSONParser.ACCEPT_TAILLING_SPACE)) != (JSONParser.ACCEPT_TAILLING_DATA
```

Comment:
my screen is not large enough to read this line ...
may I get a human translation for this change?

Reply:
@UrielCh It's a style miss change. I will revert it.